manifests/plugin/disk.pp

```diff
@@ -15,7 +15,7 @@
   validate_bool($ignoreselected)
 
   if $::osfamily == 'RedHat' {
-    if $manage_package {
+    if $manage_package != undef {
```

Comment:
may as well.  It'd guard against people using the string 'false' (which is true!).
If you're feeling really generous, a spec test would be great, but don't worry if you haven't got the time.  Thanks for spotting this bug!